src/components/books/BookDetails.vue

```diff
@@ -48,7 +48,7 @@
     </div>
     <div data-cy="book-description">
       <vue-clamp
-        v-if="hasDescription"
+        v-if="hasDescription && item.description"
```

Comment:
Could you clarify why do we need this change in particular?

Reply:
good catch, we don't actually need it, I left it by accident. It was one of my exploratory changes. I'll remove it.

Reply:
Got it, thank you 👍🏻